### ISSUE TITLE
Downtime history shows incorrect data

### ISSUE THREAD

**Issue by SlavikCA** [elastic/uptime#202]
**Describe the bug**
I have one monitor, which is down for a couple hours (~9 hours) already.
Uptime monitor status is "Down", as expected.
However, "Downtime history" shows no down history and says: "This monitor has never been down".

**To Reproduce**
My monitor config:
```
- type: http
  id: "https-ub18ovh.slavikf.com"
  name: "https://ub18ovh.slavikf.com"
  urls: [        "https://ub18ovh.slavikf.com/slavik_info.php" ]
  schedule: '@every 5m'
  timeout: 30s
  check.request.method: GET
  check.response.status: 200
  check.response.body: "PHP Version 7.2"
```

I'm using Elastic Cloud version 7.7 of Elastic and Kibana.
The heartbeat is installed on Ubuntu and is version 7.7.
The Uptime app is opened with Chrome 81.0.

![Screen Shot 2020-05-14 at 11 23 41 AM](https://user-images.githubusercontent.com/6293679/81971356-cc840c00-95d5-11ea-9e0c-63bfd2f8d642.jpg)

**Comment by SlavikCA** [comment]
And here is the detailed view of the same monitor:

![Screen Shot 2020-05-14 at 11 29 30 AM](https://user-images.githubusercontent.com/6293679/81971638-40beaf80-95d6-11ea-9a37-255d66bed8ff.jpg)

**Comment by shahzad31** [comment]
Hi @SlavikCA thanks for reporting the problem, we have already fixed it in 7.8 https://github.com/elastic/kibana/issues/64017 
We will evaluate now, if we want to backport it to 7.7.x.

Thank you

**Comment by andrewvc** [comment]
Closing since this is already merged. We should try to backport to 7.7.2
